basics-assignment-1-start/basics-assignment-1-start/src/app/conrol-dictate/conrol-dictate.component.html

```diff
@@ -0,0 +1,30 @@
+
+<button (click)="changeAlerts()">Change Alerts</button>
+
+<mat-card *ngIf="selectedAlert" class="custom-card">
```

Comment:
why did you add the css class?

Reply:
i just try somthing to add with the css and i forgot to delete